cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -209,13 +210,39 @@ pub type PoolFungiblesTransactor = FungiblesAdapter<
 	CheckingAccount,
 >;
 
+parameter_types! {
+	/// Taken from the real gas and deposits of a standard ERC20 transfer call.
+	pub const ERC20TransferGasLimit: Weight = Weight::from_parts(700_000_000, 100_000);
```

Comment:
Contracts can be up to 100k in size. So using a max sized contract would not fit into this limit. Contracts using less code and more data can be even bigger in size. I suggest adding some head room here.

```suggestion
	pub const ERC20TransferGasLimit: Weight = Weight::from_parts(700_000_000, 200_000);
```